duckduckgo/Android
# Remove incremental rollout test

Task/Issue URL: https://app.asana.com/0/0/1205402504886433/f

### Description
Remove irrelevant incremental rollout test

### Steps to test this PR
QA-Optional

If you want to sanity test:
- Install develop version
- Open a website
- Hide tips
- Restart app
- Install from this build
- Restart app
- Verify no crash
